--- xorp-1.4/configure.in	2007-03-14 19:59:04.000000000 +0100
+++ xorp-1.4/configure.in	2007-03-26 20:20:59.000000000 +0200
@@ -875,20 +875,20 @@ dnl GCC-specific warning flags
 dnl -----------------------------------------
 
 if test "${GCC}" = "yes" ; then
-	dnl
-	dnl Filter-out the "-g" flag
-	dnl
-	CFLAGS=`echo $CFLAGS |  sed 's/-g//g'`
-	CXXFLAGS=`echo $CXXFLAGS |  sed 's/-g//g'`
-
-	dnl
-	dnl If debugging is enabled, try to add the "-g" flag
-	dnl
-	if test "${enable_debug}" = "yes" ; then
-		XR_TRY_ADD_CFLAGS("-g")
-		XR_TRY_ADD_CXXFLAGS("-g")
-	fi
-
+#	dnl
+#	dnl Filter-out the "-g" flag
+#	dnl
+#	CFLAGS=`echo $CFLAGS |  sed 's/-g//g'`
+#	CXXFLAGS=`echo $CXXFLAGS |  sed 's/-g//g'`
+#
+#	dnl
+#	dnl If debugging is enabled, try to add the "-g" flag
+#	dnl
+#	if test "${enable_debug}" = "yes" ; then
+#		XR_TRY_ADD_CFLAGS("-g")
+#		XR_TRY_ADD_CXXFLAGS("-g")
+#	fi
+#
 	dnl
 	dnl TODO: add -ansi -pedantic
 	dnl XXX: Flag -Wtraditional is not recommended for C, because:
